src/components/Validator.js

```diff
@@ -47,9 +47,9 @@ export const Validator = {
       }
     });
 
-    const customErrorMessage = _.get(component, 'component.validate.customMessage');
```

Comment:
Doesn't this break reverse compatibility?

Reply:
I've added back in the 'component.validate.customMessage' to handle reverse compatibility. 

component.validate.custom is the entire string object for example: 
  "valid = (input === 3) ? true : 'Must be 3'" 

where 'result' is the actual string value. therefore 'result' needs to be translated.

Reply:
But you are not executing the value to get the result... you are basically translating ```valid = (input === 3) ? true : 'Must be 3'``` instead of ```Must be 3```. I don't see how this is working the way you expect.

Reply:
That's not right. Otherwise what is the point of ```customErrorMessage``` if it is never used or being ignored? This breaks reverse compatibility. Please fix.